### ISSUE TITLE
feedback protection for [knob] and other GUIs 

### ISSUE THREAD

**Issue by tomara-x** [porres/pd-else#1638]
setting the same symbol for send and receive causes funny stuff to happen :(

**Comment by porres** [comment]
Yeah, I don't believe in feedback protection

**Comment by tomara-x** [comment]
i don't do feedback protection because society says i have to

**Comment by porres** [comment]
I'm actually serious though, no object in ELSE has it and I don't intend to offer it. I also think the feedback filter in Vanilla is badly coded and has some issues and is a bit pointless

**Comment by tomara-x** [comment]
still some way of making it not crash would be nice

**Comment by porres** [comment]
how does it crash?

**Comment by tomara-x** [comment]
is that still a question? i'm guessing you probably tested it

**Comment by porres** [comment]
yeah, I was away from keyboard when I asked it and I eventually tested it. It freezes because of some infinite loop situation. I'll do the same as what Miller does for the atom boxes, whatch for it and print an error.

**Comment by porres** [comment]
done with https://github.com/porres/pd-else/commit/d1ce4300418e8c9ff028906e24595308851afbed
